src/Internal.AspNetCore.Sdk/build/Git.targets

```diff
@@ -120,4 +127,13 @@ Generates an assembly attribute with commit hash
     <Message Importance="normal" Text="Generated commit hash attribute file $(GeneratedCommitHashAttributeFile)" />
   </Target>
 
+  <Target Name="_AppendRepositoryCommitUrlToPackageDescription"
+    DependsOnTargets="ResolveCommitHash"
+    Condition="'$(GenerateRepositoryCommitUrlInDescription)' == 'true' AND '$(RepositoryUrl)' !='' AND '$(RepositoryCommit)' != ''">
+    <PropertyGroup>
+      <PackageDescription>$(PackageDescription)
+Repository commit url: $(RepositoryUrl)/tree/$(RepositoryCommit)</PackageDescription>
```

Comment:
I like the wording roslyn uses. https://www.nuget.org/packages/Microsoft.CodeAnalysis/

![image](https://user-images.githubusercontent.com/2696087/40383138-fb04dcb8-5db4-11e8-9d23-15c0ad925831.png)

"This package was built from the source at (url)" What do you think?

Reply:
What about the assembly metadata? SourceCommitUrl?

Reply:
The don't embed the URL into the assembly, just the commit hash.

![image](https://user-images.githubusercontent.com/2696087/40389437-4b75c330-5dc7-11e8-8ac3-acab5ba4dbf4.png)

Reply:
@davidfowl wants both: https://github.com/aspnet/BuildTools/pull/678#discussion_r189793496

Reply:
I don't care either way about the assembly attributes. Those are almost never visible to the user anyways.